src/compiler/crystal/compiler.cr

```diff
@@ -604,18 +591,50 @@ module Crystal
       end
     end
 
-    private def process_wrapper(command, args = nil, &)
+    private def run_linker(linker_name, command, args)
       print_command(command, args) if verbose?
 
-      status = yield command, args
+      begin
+        Process.run(command, args, shell: true,
+          input: Process::Redirect::Close, output: Process::Redirect::Inherit, error: Process::Redirect::Pipe) do |process|
+          process.error.each_line(chomp: false) do |line|
+            next if line.matches?(/^--: 1: .*: not found$/)
```

Comment:
Should we avoid skipping lines if --verbose is used?

Reply:
Sounds good. This is definitely one of the more fuzzy aspects. I suppose it should also only skip the *first* line, not if it appears later in the output.

Reply:
I don't follow why only the first line. I would avoid filtering entirely on --verbose.

Reply:
The filter is supposed to remove the "command not found" string? The `zsh: command not found: xxxx`? Or is it something else?

Reply:
Yeah, that's the idea. But, I think it might be better to remove that. At least for now. It's a convenience feature, but not really necessary. And it's not super accurate in any case.